validator/testcases/regex.py

```diff
@@ -2901,9 +2915,34 @@ def munge_filename(name):
                         "the `%s` preference branch" % branch)))
         for branch, reason in BANNED_PREF_BRANCHES))
 
-STRING_REGEXPS += PREF_REGEXPS
+# For tests in literal strings, add help text suggesting passing the
+# preference directly to reference getter functions.
+PREF_STRING_HELP = (
+    "If you are reading, but not writing, this preference, please consider "
+    "passing a string literal directly to `Preferences.get()` or "
+    "`nsIPrefBranch.get*Pref`.")
+
+
+def maybe_tuple(value):
+    if isinstance(value, tuple):
+        return value
+    return (value,)
```

Comment:
If we handled a third case for some other type, I'd write the second conditional as a second `if` rather than an `elif`. Only I'd actually probably just make it `isinstance(value, (list, tuple))`.

In any case, I don't really have a strong opinion here, but I generally lean toward the `else`-less form, since the `else` is superfluous, and doesn't really make anything clearer.